habitat-lab/habitat/utils/visualizations/utils.py

```diff
@@ -220,6 +220,13 @@ def observations_to_image(observation: Dict, info: Dict) -> np.ndarray:
             obs_k = observation[sensor_name]
             if not isinstance(obs_k, np.ndarray):
                 obs_k = obs_k.cpu().numpy()
+            if (
+                sensor_name == "cat_nav_goal_segmentation"
+                and obs_k.shape[2] == 2
+            ):
+                obs_k = np.clip(obs_k[:, :, 0] + obs_k[:, :, 1] * 0.2, 0, 1)[
```

Comment:
What is happening here?

Reply:
Visualizing the 2-channel goal segmentation sensor by combining the 2 channels into 1. The target object will be white and the receptacle is gray.

Reply:
Thanks for explaining!